### DISCUSSION TITLE
Currency Change

### DISCUSSION THREAD

**Discussion by gitvivek14** [evershopcommerce/evershop#649] [Feature Requests & Ideas]
Is there a way so that I can change the currency

**Comment by RobbieWoodhams** [comment]
You need to add a config folder in the root directory and then add a default.json file otherwise evershop uses the default theme. This is whats in mine:
{
    "shop": {
        "currency": "GBP",
        "language": "en-GB",
        "weightUnit": "kg"
    },
    "system": {
        "theme": "default"
    }
}

**Reply by gitvivek14** [reply]
Already figured that out , but anyways thanku so much @RobbieWoodhams .
